docs/sources/flow/reference/components/prometheus.exporter.self.md

```diff
@@ -1,57 +1,57 @@
 ---
 aliases:
-- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/prometheus.exporter.agent/
-- /docs/grafana-cloud/send-data/agent/flow/reference/components/prometheus.exporter.agent/
-canonical: https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.exporter.agent/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/prometheus.exporter.self/
+- /docs/grafana-cloud/send-data/agent/flow/reference/components/prometheus.exporter.self/
```

Comment:
```suggestion
- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/prometheus.exporter.agent/
- /docs/grafana-cloud/send-data/agent/flow/reference/components/prometheus.exporter.agent/
./prometheus.exporter.agent/
```

According to: https://grafana.com/docs/writers-toolkit/write/front-matter/#aliases the alias should be the original URL not the new URL.  I'd leave these as they were. and add one new....

We should do a `make docs` to make sure the alias is working as intended.

Reply:
(For context, `make docs` is expected to be run from within the `docs/` folder; it doesn't work at the root)